src/cluster.bootstrap/Akka.Management.Cluster.Bootstrap/Internal/BootstrapCoordinator.cs

```diff
@@ -164,6 +165,8 @@ public BootstrapCoordinator(ServiceDiscovery discovery, IJoinDecider joinDecider
                 _settings.ContactPointDiscovery.EffectiveName(Context.System),
                 _settings.ContactPointDiscovery.PortName,
                 _settings.ContactPointDiscovery.Protocol);
+
+            _fallbackPort = _settings.ContactPoint.FallbackPort ?? AkkaManagement.Get(Context.System).Settings.Http.Port;
```

Comment:
We read the actual settings value directly from the plugin, not the plugin HOCON config